onadata/apps/api/viewsets/open_data_viewset.py

```diff
@@ -132,42 +132,48 @@ def data(self, request, **kwargs):
                 qs_kwargs.update({'id__gt': gt_id})
 
             instances = Instance.objects.filter(**qs_kwargs).order_by('pk')
-            length = self.total_count = instances.count()
+            self.total_count = instances.count()
```

Comment:
in this file, `total_count` is only referred to on line 138, can we inline this on line 138 and remove it here so that we only do this work when `count` is `True`?

Reply:
There is a mixin that relies on self.total_count hence it being set here is used elsewhere in the code.

Reply:
I see, let's remove that mixin, it looks like the only place X-Total is used is in the JS in this repo